WordPress/Classes/Utility/Analytics/WPAnalytics+Domains.swift

```diff
@@ -53,3 +61,9 @@ enum DomainsAnalyticsWebViewOrigin: String {
     case siteCreation = "site_creation"
     case menu
 }
+
+// TODO: remove when WPAppAnalyticsTests get rewritten, preferably in Swift
+@objc final class WPAnalyticsTesting {
```

Comment:
I didn't want to mess with the Objective-C code too much 😞 
It would make sense to refactor it together with a Swift rewrite.